### PR TITLE
Added logz.io for log shipping

### PR DESCRIPTION
**Pull request by jsdoe**
Hi there,

I added support for logz.io through their corresponding plugin.

Three ENV vars need to be considered:
- FLUENT_LOGZIO_URL: The endpoint to use. Typically https://listener.logz.io:8071
- FLUENT_LOGZIO_TOKEN: Your logz.io access token
- FLUENT_LOGZIO_TYPE: The logz.io [log type](https://support.logz.io/hc/en-us/articles/209486049-What-is-type-and-why-do-I-need-it-)

### CODE REVIEW

File: docker-image/v0.12/alpine-logzio/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+RUN set -ex \
+    && apk update && apk upgrade \
+    && apk add --no-cache --virtual .build-deps \
+        build-base \
+        ruby-dev \
+        libffi-dev \
+    && echo 'gem: --no-document' >> /etc/gemrc \
+    && gem install fluent-plugin-secure-forward \
+    && gem install fluent-plugin-record-reformer \
+    && gem install fluent-plugin-logzio \
```
Comment (cosmo0920) [inline docker-image/v0.12/alpine-logzio/Dockerfile]: fluent-plugin-logzio v0.0.15 or newer is not compatible for Fluentd v0.12.
Could you specify fluent-plugin-logzio version explicitly?

```
&& gem install fluent-plugin-logzio -v 0.0.14 \
```

ref: https://github.com/logzio/fluent-plugin-logzio/tree/db20fd0887f15ebfb1ba5b241fb308d0fc2454d7#requirements

File: docker-image/v0.12/debian-logzio/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+RUN buildDeps="sudo make gcc g++ libc-dev ruby-dev libffi-dev" \
+     && apt-get update \
+     && apt-get upgrade -y \
+     && apt-get install \
+     -y --no-install-recommends \
+     $buildDeps \
+    && echo 'gem: --no-document' >> /etc/gemrc \
+    && gem install fluent-plugin-secure-forward \
+    && gem install fluent-plugin-record-reformer \
+    && gem install fluent-plugin-logzio \
```
Comment (cosmo0920) [inline docker-image/v0.12/debian-logzio/Dockerfile]: ditto.

**Comment by jsdoe** [comment]
@cosmo0920 sure, done

**Comment by edsiper** [comment]
does 97f47df already implement this properly ?

**Comment by cosmo0920** [comment]
We already provided logzio docker image.
This should be closed.
Anyway, thank you for requesting to add logzio image.

**Pull request closed without merging**